.github/workflows/add-word.yaml

```
@@ -42,7 +42,9 @@ jobs:
           branch: add-word-${{ inputs.word }}
           title: "feat(words): add ${{ inputs.word }}"
           commit-message: "feat(words): add ${{ inputs.word }}"
-          body: "TODO: Update this comment and add some reference links."
+          body: |
+            TODO: Update this comment and add some reference links.
+            Please check [CONTRIBUTING.md](https://github.com/tier4/autoware-spell-check-dict/CONTRIBUTING.md) before sending PR.
```

Meybe, the link for CONTRIBUTING.md is invalid.
for example, [tier4/autoware-spell-check-dict/README.md](https://github.com/tier4/autoware-spell-check-dict/README.md) is invalid link too.

```suggestion
            Please check [CONTRIBUTING.md](https://github.com/tier4/autoware-spell-check-dict/blob/main/CONTRIBUTING.md) before sending PR.
```

---

Thanks!!
Updated in https://github.com/tier4/autoware-spell-check-dict/pull/763/commits/3c5dd68687723386be9490d279a6d75d0f8aeffc